### DISCUSSION TITLE
Grafana password update from 1.25 to 1.25.2

### DISCUSSION THREAD

**Discussion by regator99** [teslamate-org/teslamate#2344] [General]
Saw the release notes about the default password is to be used after the update (admin/admin), however, that is not working for me.

I am using the default install (local network only - docker on Synology), I have tried my other passwords but to no avail.  I have searched the docker-compose.yml file on my Synology but no clues.  Any advice on how to get into Grafana.  

Looking at my container I see the environment variables and it appears that admin/admin are still there but I am a novice to docker.

![Capture](https://user-images.githubusercontent.com/11617981/150170907-4dfd56eb-98c6-43b4-bd7f-e7aeb887534f.PNG)

**Comment by cwanja** [comment]
There was similar issue in #2329, however the _default_ password did not change. Adrian also commented to that extent. I posted in that issue a URL in how to change the Grafana password via the docker CLI.

**Reply by regator99** [reply]
Thank you, reseting password worked for me
